code/controllers/subsystem/vote.dm

```diff
@@ -206,9 +206,7 @@ SUBSYSTEM_DEF(vote)
 		if(vote_initiator)
 			to_chat(vote_initiator, span_warning("Invalid voting choice."))
 		return FALSE
-		
-	if(!to_vote.player_startable)
-		return FALSE
```

Comment:
Code changes outside the `/monkestation/` folder need to be marked appropriately. See <https://github.com/Monkestation/Monkestation2.0/blob/master/monkestation/README.md> for more information.